### ISSUE TITLE
Cannot add item to cart, Unsupported operand types

### ISSUE THREAD

**Issue by muzhawir** [bagisto/bagisto#9475]
### Terms

- [X] Before you openning this issue, i have checked if the issue has already been reported.

### Bagisto Version(s) affected

2.0.0

### Issue Description

When I try to click the "Add to Cart" button, the item cannot be added to the cart. The debug bar prints the log `"[17:10:28] LOG.error: Unsupported operand types: string * int {"exception": {}}"`

### Preconditions

- Running on Laravel 10 using Sail docker
- Using guest user

### Steps to reproduce

1. Go to home page
2. Hover on the product item 
3. Click "Add To Cart" Button

### Expected Result

Item added to cart 

### Actual Result

- Item cannot added to cart

### Screenshots

![error](https://github.com/bagisto/bagisto/assets/7391703/a40ff67a-a1f0-4b01-a7ee-7193f89a5cda)

**Comment by shivendra-webkul** [comment]
Hi @muzhawir ,

Thank you for your attention to this matter. I wanted to inform you that the reported issue has not been addressed.
If you have any further questions or encounter any other issues, feel free to reach out. Your cooperation is much appreciated.

**Comment by emilrustamov** [comment]
Did u fix it? I have the same problem
![image](https://github.com/user-attachments/assets/0d59f96f-0d56-4abd-a66a-83a938629de1)

**Comment by muzhawir** [comment]
I fixed this issue some time ago. The problem arises because the first operand is a string while the second operand is an int. This type mismatch prevents the mathematical operation from being performed. Therefore, the first operand needs to be cast to an int.

![image](https://github.com/user-attachments/assets/f1cb391c-6900-4d0c-956d-ffcb2ffe02ab)

I have now opened a Pull Request for this in #10153
